src/types/Token.ts

```diff
@@ -1,10 +1,19 @@
-interface Token {
+type CustomTokenPart<T> = {
+  [P in keyof T]: T[P]
+};
+
+type StaticTokenPart = {
   access_token: string;
   expires_in?: number;
   scope?: string[];
   token_type?: string;
   local_expiry?: number;
-  [key: string]: any;
 }
 
-export { Token };
+type Token<T = any> = CustomTokenPart<T> & StaticTokenPart;
```

Comment:
@bzums There is no difference, `CustomTokenPart<T>` was created during my analysis of the requirements and should be replaced with `T`. 

What about the default type of the generic parameter? Should the consumer be forced to give a certain type or should we leave it as it is?